src/main/java/seedu/address/model/poc/Phone.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.poc;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Company's phone number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
+ */
+public class Phone {
```

Comment:
Why is the class name for this class `Phone` and not `PersonPhone` while the class name for `Name` is `PersonName`? Should they be standardised?